### PR TITLE
Build rework and Linux is finally completely back to life :)

### PR DESCRIPTION
**Pull request by okawo80085**
List of changes:
* Fixed display component on Linux(see openvr issue [#1583](https://github.com/ValveSoftware/openvr/issues/1583#issue-981808426) and [8f1814e](https://github.com/okawo80085/hobo_vr/commit/8f1814e0f1bd71bb999297c62056528cf1d866e6))
* Fixed driver build(see [53ada3e](https://github.com/okawo80085/hobo_vr/commit/53ada3e2f0111c2646b8ead3e62fe4089674a0f8))
* Fixed a small bug in the offline Linux installer build script
* Changed default udu settings for the driver to none, making it the poser's responsibility to register it's devices with a `Emsg_uduString` manager message(see [manager protocol](https://github.com/okawo80085/hobo_vr/wiki/Driver:-communication-protocol#manager-protocol))
* Minor changes in readmes

I mostly tested it, but i still need review and more testing by someone who is not me

The changes in this branch are up for a release after it's merged

### CODE REVIEW

File: driver/README.md [hunk 1]
```diff
@@ -3,49 +3,47 @@ this is only the source and building instruction for the `hobovr` driver, the bu
...
 cmake --build build64 --config Release
-```
-[more info on these cmake commands](https://stackoverflow.com/a/28370892/10190971)
 
-now after the build is done move the 32 bit binary into `../hobovr/bin/win32` and the 64 bit binary into `../hobovr/bin/win64`
+cmake --install build32
+cmake --install build64
+```
 
 congrats, you're done, now when you start steamvr it'll load your build of `hobovr`
```
Comment (SimLeek) [inline driver/README.md:29]: Well, if driverregister has been run and the location was set, this is true.

File: driver/README.md [hunk 2]
```diff
@@ -3,49 +3,47 @@ this is only the source and building instruction for the `hobovr` driver, the bu
...
-```
+cmake --build build32 --config Release
+cmake --build build64 --config Release
 
-then rename `libdriver_hobovr.so` to `driver_hobovr.so` and copy it to both `../hobovr/bin/linux32` *and* `../hobovr/bin/linux64`
+cmake --install build32
+cmake --install build64
+```
 
 congrats, you're done, now when you start steamvr it'll load your build of `hobovr`
```
Comment (SimLeek) [inline driver/README.md:49]: same thing about driverregister here

**Comment by SimLeek** [comment]
Looked at the code. Looks pretty good. Now I just gotta follow the build instructions and see if the binaries work.

File: driver/README.md (hunk 1 above)
Comment (okawo80085) [reply driver/README.md:29]: U have a point about registering the driver, will adding that to pre build steps work?

**Comment by okawo80085** [comment]
Now that I look at the branch, I may have forgot to update a few readmes, mentioning adddriver in the build instructions is the worst one I think

File: driver/README.md (hunk 1 above)
Comment (SimLeek) [reply driver/README.md:29]: Ah, yea, that'd work too

**Comment by okawo80085** [comment]
Fixed i think? can you check it again @SimLeek

**Review by okawo80085 — COMMENTED** [review]
Not mentioned in the [d705b25](https://github.com/okawo80085/hobo_vr/pull/43/commits/e06ff7e4b67a732b093e00a332556c7b7d297ceb) commit message is the change in extended display component settings, default resolution is set to 1080p and `IsDisplayReal` is set to false

**Comment by okawo80085** [comment]
So i finally got to my Linux patches, boiiiiiiiiiii they're dumb, it honestly feels like SteamVR behaves differently on Linux compared to Windows, so... we'll still have some bugs on Linux which are out of our control, due to them being a SteamVR issue, like multi driver interactions on Linux, our devices just randomly decide to not show up on Linux while on Windows they show up fine, with seemingly exactly the same driver...

**Comment by okawo80085** [comment]
@SimLeek is there anything that doesn't look right to you?

**Comment by SimLeek** [comment]
Looks good. 👍

**Comment by okawo80085** [comment]
Sweet

**Pull request merged**